docs/development/custom-ca-citadel/README.md

```diff
@@ -0,0 +1,104 @@
+# Custom CA *(intermediate one)* to be used by Citadel
+
+## Background
+
+Imagine to work for a big company owning its own PKI. Most of this companies owns its own Root CA generating
+intermediate CAs to sign Clients and/or Server Certificates.
+
+As Istio provides a way to issue certificates *(by default, self-signed)* for every workload inside a mesh,
+it has to be a way to provide a custom CA to issue workload certificates instead of using a *(default)* new one.
+
+
+## Requirements
+
+Ensure to have the following files:
+
+- `certs/ca-cert.pem`: Certificate of the Intermediate CA.
+- `certs/ca-key.pem`: Private key of the Intermediate CA.
+- `certs/root-cert.pem`: Certificate of the Root CA.
+- `certs/cert-chain.pem`: Certificate Chain. `cat certs/ca-cert.pem certs/root-cert.pem > certs/cert-chain.pem`.
+
+Also, it's required to know what `katalog/istio/sidecar-injection/configuration` it configured because it
+needs to be modified.
+
+### Hands on
+
+#### Istio Sidecar Configuration
+
+Look in the `kustomization.yaml` file. It has to be defined a `service-mesh/istio/sidecar-injection/configuration/*`
+istio configuration base.
+
+*Example Istio `kustomization.yaml` file:*
+
+```yaml
+bases:
+  - ../vendor/katalog/service-mesh/istio/minimal
+  - ../vendor/katalog/service-mesh/istio/telemetry
+  - ../vendor/katalog/service-mesh/istio/citadel
+  - ../vendor/katalog/service-mesh/istio/egress-gateway
+  - ../vendor/katalog/service-mesh/istio/sidecar-injection
+  - ../vendor/katalog/service-mesh/istio/sidecar-injection/configuration/sidecar-injection-and-egress
+```
+
+This installation uses `sidecar-injection-and-egress` istio configuration file.
+Override the `selfSigned` attribute with `false` value copying to the new kustomize project:
```

Comment:
should I override the attribute or the value?

Reply:
With the sed expression you will find after that sentence you will override the `attribute:value` pair!

Reply:
Now I understand what you intended, what do you think about refrasing that sentence to:
Override the `selfSigned` attribute's value from `true` to `false` a save it to the new kustomize project: